pom.xml

```diff
@@ -98,7 +98,12 @@
         <dependency>
             <groupId>com.graphql-java</groupId>
             <artifactId>graphql-java</artifactId>
-            <version>16.2</version>
+            <version>17.3</version>
```

Comment:
@kurenchuksergey Can you consider bumping this to `17.4` to include changes that resolves a [recent vulnerability](https://github.com/advisories/GHSA-v62j-cxhh-fq22)?